inference-engine/thirdparty/clDNN/src/layout_optimizer.cpp

```diff
@@ -237,8 +237,13 @@ bool layout_optimizer::can_fuse_reorder_to_prev(program_node& prev, program_node
         return true;
 
     if (prev.is_type<permute>()) {
+        auto& permute_order = prev.as<permute>().get_primitive()->permute_order;
```

Comment:
Yes seems that #5761 includes similar thing for permute reorder fusing. However for both #5761 and this PR, I'd like to  suggest below condition. If we apply #5761 no [fsv permute => bfyx reorder] will be fused. I think the following case is the current porblem 
- **permute_ref** fsv => fsv + reorder bfyx, and the f axis is going to the last dimension. 
- This case, f axis is aleady stored in a continuous way for fsv size, so fusing reorder is not beneficial. 
- However for tiled kernel, fusing will be still beneficial. Tiled kernel is used when the permute is bfyx => byxf.

Reply:
@yeonbok could you discuss it with @lznamens and define the best solution and decide how it will be merged (within #5761 or in this PR)? thanks

Reply:
@lznamens  @vladimir-paramuzov  Whether this permute is handled in this PR or #5761,  I'd like to use the following policy to pervent permute reorder fusing 

-It is [permute with blocked format , fsv =>fsv] => [reorder to non blocked , i.e., bfyx]
- && permute_order[1] == 2   // Moving f axis to the last dim (because f is alreaedy continuous) 
- && the permute is "not rotating case (bfyx => byxf) " // rotating cases are good to fuse reorder becuase it is mapped to tiled kernel

Reply:
@yeonbok I've checked your patch and it works better than my solution. I've got results from dlbenchmark and there are several minor regressions for permute + reorder cases (I tested it before only locally on hrnet and my logic worked good for this case). I can revert my last commit (related to permute) and replace it with you patch and after that restart dlbenchmark test with your changes. Do you agree?

Reply:
No problem. Then could you please cherry pick our commit to your branch?

Reply:
Yes, thanks.

Reply:
Cherry-pick and perf test via dl-benchmark have been done, no regressions detected, please review here: 
https://github.com/openvinotoolkit/openvino/pull/5761